sections/01_abstract.md

```diff
@@ -1,3 +1,18 @@
 ## Abstract
 
-Abstract goes here.
+Deep learning, a new class of machine learning algorithms, has recently showed
+impressive results across a variety domains. Biomedicine and genomics, both
+data- and feature-rich and yet complex and ill-understood, presents an obvious
+target for applying this new approach. We examine applications of deep learning
+to a variety of biomedical data problems - in patient classification,
+fundamental research and patient treatment -  to determine if similar progress
+can be made here or what unique challenges the biomedical sphere holds. We find
+that while deep learning has achieved (sometimes remarkable) advances on prior
+state-of-the-art methods, it has not yet revolutionized or definitively solved
+any problem. In some cases, this is probably simply a matter of time but
```

Comment:
Bit of a tough one this, especially considered the tight length restrictions. Anyway, I've had a go.